src/layout/app.css

```diff
@@ -26,6 +26,10 @@ html, body {
   margin: 0;
 }
 
+.noticebox-wrapper{
+  margin-right: 15px;
```

Comment:
I think it would make more sense to use logical properties when adding new css, so `margin-inline-end` here